src/js/letters/utils/constants.js

```diff
@@ -4,10 +4,76 @@ export const BACKEND_SERVICE_ERROR = 'BACKEND_SERVICE_ERROR'; // 503 or 504
 export const INVALID_ADDRESS_PROPERTY = 'INVALID_ADDRESS_PROPERTY'; // 422
 export const GET_LETTERS_FAILURE = 'GET_LETTERS_FAILURE';
 export const GET_LETTERS_SUCCESS = 'GET_LETTERS_SUCCESS';
+export const GET_ADDRESS_FAILURE = 'GET_ADDRESS_FAILURE';
+export const GET_ADDRESS_SUCCESS = 'GET_ADDRESS_SUCCESS';
 export const GET_BENEFIT_SUMMARY_OPTIONS_FAILURE = 'GET_BENEFIT_SUMMARY_OPTIONS_FAILURE';
 export const GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS = 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS';
 export const GET_LETTER_PDF_FAILURE = 'GET_LETTER_PDF_FAILURE';
 export const GET_LETTER_PDF_SUCCESS = 'GET_LETTER_PDF_SUCCESS';
 export const LETTER_ELIGIBILITY_ERROR = 'LETTER_ELIGIBILITY_ERROR';
 export const UPDATE_BENFIT_SUMMARY_REQUEST_OPTION = 'UPDATE_BENFIT_SUMMARY_REQUEST_OPTION';
 export const UPDATE_ADDRESS = 'UPDATE_ADDRESS';
+
+export const STATE_CODE_TO_NAME = {
+  AL: 'Alabama',
+  AK: 'Alaska',
+  AZ: 'Arizona',
+  AR: 'Arkansas',
+  CA: 'California',
+  CO: 'Colorado',
+  CT: 'Connecticut',
+  DE: 'Delaware',
+  FL: 'Florida',
+  GA: 'Georgia',
+  HI: 'Hawaii',
+  ID: 'Idaho',
+  IL: 'Illinois',
+  IN: 'Indiana',
+  IA: 'Iowa',
+  KS: 'Kansas',
+  KY: 'Kentucky',
+  LA: 'Louisiana',
+  ME: 'Maine',
+  MD: 'Maryland',
+  MA: 'Massachusetts',
+  MI: 'Michigan',
+  MN: 'Minnesota',
+  MS: 'Mississippi',
+  MO: 'Missouri',
+  MT: 'Montana',
+  NE: 'Nebraska',
+  NV: 'Nevada',
+  NH: 'New Hampshire',
+  NJ: 'New Jersey',
+  NM: 'New Mexico',
+  NY: 'New York',
+  NC: 'North Carolina',
+  ND: 'North Dakota',
+  OH: 'Ohio',
+  OK: 'Oklahoma',
+  OR: 'Oregon',
+  PA: 'Pennsylvania',
+  RI: 'Rhode Island',
+  SC: 'South Carolina',
+  SD: 'South Dakota',
+  TN: 'Tennessee',
+  TX: 'Texas',
+  UT: 'Utah',
+  VT: 'Vermont',
+  VA: 'Virginia',
+  WA: 'Washington',
+  WV: 'West Virginia',
+  WI: 'Wisconsin',
+  WY: 'Wyoming',
+  AS: 'American Samoa',
+  DC: 'District of Columbia',
+  FM: 'Federated States of Micronesia',
+  GU: 'Guam',
+  MH: 'Marshall Islands',
+  MP: 'Northern Mariana Islands',
+  PW: 'Palau',
+  PR: '',
```

Comment:
@raquelromano-gov do you still have the email with the rest of the translations for these states? Thanks!

Reply:
Should be here https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4110#issuecomment-329359875 but yes, I can make sure you are all on that thread (later tonight)

Reply:
Perfect, thanks!